x-pack/solutions/security/plugins/security_solution/public/notes/api/api.ts

```diff
@@ -28,6 +28,10 @@ export const createNote = async ({ note }: { note: BareNote }) => {
     });
     return response.note;
   } catch (err) {
+    if (err?.body?.message) {
+      throw new Error(err.body.message);
+    }
+
     throw new Error(('message' in err && err.message) || 'Request failed');
```

Comment:
It feels to me that - looking at the server side code - this could actually happen... Is there a value in keeping it? If we want to keep it just in case, couldn't we merge the if condition and this together?

Reply:
the first one is actually different in that it assumes the error originates on the backend and has a specified body, the other case can be actually anything on the client side

Reply:
done